spec/requests/orders_spec.rb

```diff
@@ -10,5 +10,20 @@
       expect(response).to have_http_status(200)
       expect(orders.length).to eq 1
     # pending "add some examples (or delete) #{__FILE__}"
+    end
+  end
+  describe "POST /create" do
+    it "creates a order" do
+      user = User.where(email: "test@example.com").first_or_create(password: "password", password_confirmation: "password")
+      order_params = { 
+        order:{
+          user_id: 1,
+          product_id: 1,
+      }}
+      post '/orders', params: order_params
+        expect(response).to have_http_status(200)
+        order = order.first
+        expect(order.user_id).to eq 1
```

Comment:
Is this test passing?  Do you need a product to have been created like user is on line 17 for orders?